### PR TITLE
Fix Step 3a of TC_xxxCONC_1_1

### LINKED ISSUE #28838: Fix Step 3a of TC_xxxCONC_1_1
### Description

As part of #28361, Step 3a of `TC_xxxCONC_1_1` was edited to check for 0 if no feature flags were set, e.g:

```
    - label:
          "Step 3a: Read the global attribute: FeatureMap and check for either
          bit 0 or 1 set"
      PICS:
          " !OZCONC.S.F00 && !OZCONC.S.F01 && !OZCONC.S.F02 && !OZCONC.S.F03 &&
          !OZCONC.S.F04 && !OZCONC.S.F05"
      command: "readAttribute"
      attribute: "FeatureMap"
      response:
          value: 0
          constraints:
              type: bitmap32
```

Technically, this is incorrect, as a scenario where no feature PICS are set is not a valid one. 

The test is trying to cover the fact that the `MEA` feature and the `LEV` feature have `O.a+` conformance. 

This is covered in the Test Plan via the statement: 

> At least one of bit 0 and bit 1 SHALL be 1.

So, this test step needs to be reverted to test for that.

### PR DESCRIPTION
**Pull request by mhazley**
### Description

Fixes test step 3a in all Concentration Measurement 1.1 tests. 

Before it was checking for a non-compliant state. 

Now, it checks for the statement in the Test Plan: 

> At least one of bit 0 and bit 1 SHALL be 1.

### Links

Fixes #28838

### CODE REVIEW

**Comment by mhazley** [comment]
@manjunath-grl it would be great if you could review, I think this is reverting your change!

**Review by manjunath-grl — APPROVED** [review]

File: src/app/tests/suites/certification/Test_TC_CDOCONC_1_1.yaml [hunk 1]
```diff
@@ -42,15 +42,12 @@ tests:
...
-      PICS:
-          " !CDOCONC.S.F00 && !CDOCONC.S.F01 && !CDOCONC.S.F02 && !CDOCONC.S.F03
-          && !CDOCONC.S.F04 && !CDOCONC.S.F05"
       command: "readAttribute"
       attribute: "FeatureMap"
       response:
-          value: 0
           constraints:
               type: bitmap32
+              hasMasksSet: [0x03]
```
Comment (cecille) [inline src/app/tests/suites/certification/Test_TC_CDOCONC_1_1.yaml:50]: Doesn't this check for both 0 and 1 set?

Comment (mhazley) [reply src/app/tests/suites/certification/Test_TC_CDOCONC_1_1.yaml:50]: It used to. Boris pointed out that the C++ version returns true for any of the bits set and we should align for Python (see #27248). 

I tested it with bit 0 set and with bit 1 set and it passes.

Comment (mhazley) [reply src/app/tests/suites/certification/Test_TC_CDOCONC_1_1.yaml:50]: FWIW, Thomas has raised an issue to make the naming better for this: https://github.com/project-chip/connectedhomeip/issues/28837

**Review by cecille — APPROVED** [review]

**Review by saurabhst — APPROVED** [review]

**Review by jmartinez-silabs — APPROVED** [review]

**Pull request merged**